### PR TITLE
feat(): Allow to change fontSize & fontFamily in editor from Settings

### LINKED ISSUE #1218: Accessibility - Users can't change editor font size
### Feature description

An accessibility issue, in the editor users should be able to change the font size and font type

### PR DESCRIPTION
**Pull request by Skraye**
close #1218
close https://github.com/kestra-io/kestra/issues/1654

### CODE REVIEW

**Comment by loicmathieu** [comment]
Small bug when you select Times
image

Also the very big input for the size is a bit strange, maybe set it to a smaller size

File: ui/src/components/settings/Settings.vue [hunk 1]
```diff
@@ -215,6 +250,39 @@
                     {value: logDisplayTypes.HIDDEN, text: this.$t("collapse all")}
                 ]
             },
+            fontFamilyOptions() {
```
[the remark this answers is not in the document]
Comment (Skraye) [reply ui/src/components/settings/Settings.vue:253]: I can check fonts, but not get a list

[the remark this answers is not in the document]
Comment (loicmathieu) [reply ui/src/components/settings/Settings.vue:253]: OK then

**Review by loicmathieu — APPROVED** [review]
LGTM

**Pull request merged**